Dockerfile

```diff
@@ -0,0 +1,23 @@
+# An image of ducktape that can be used to setup a Docker cluster where ducktape is run inside the container.
+
+FROM ubuntu:14.04
```

Comment:
parameterize ubuntu version 14.04

Reply:
It doesn't change often, but maybe later when we publish multiple OSes.